### ISSUE TITLE
Guessit series parsing error when ending in Us or Uk

### ISSUE THREAD

**Issue by gligoran** [Flexget/Flexget#2002]
### Expected behaviour:

Using `filesystem` I want to mark as collected all episodes on my disk.

A file named `Marvel's Inhumans - 1x02 - Those Who Would Destroy Us.mkv` should be parsed as series named `Marvel's Inhumans`.

### Actual behaviour:

The file mentioned above is actually parsed as `Marvel's Inhumans (US)`.

If I change the title file name to `Marvel's Inhumans - 1x02 - Those Who Would Destroy Uk.mkv` it's parsed as `Marvel's Inhumans (GB)`.

#### Config:

```
templates:
  series:paths:
    filesystem:
      regexp: '^.*/(?!\.)[^/]*?(?:avi|mkv|mp4|mpg|mov|ts)$'
      path:
        - "/path/to/tv shows/Marvel's Inhumans/"
      recursive: yes
      retrieve: files
  series:meta:
    metainfo_series: yes
    thetvdb_lookup: yes
  trakt:lookup:
    trakt_lookup:
      account: xxxxx
      username: xxxxx
  trakt:collected:no:
    if:
      - trakt_collected: reject
      - not trakt_collected: accept

tasks:
  series:mark_as_collected:
    disable: seen
    template:
      - series:paths
      - series:meta
      - trakt:lookup
      - trakt:collected:no
    list_add:
      - trakt_list:
          username: xxxxx
          account: xxxxx
          list: collection
          type: episodes
```

#### Log/Error:

```
2017-11-04 17:03 DEBUG    parser_guessit series:mark_as_collected Parsing series: `Marvel's Inhumans - 1x02 - Those Who Would Destroy Us` [options: {'identified_by': u'auto', 'name': None, 'allow_seasonless': False}]
2017-11-04 17:03 DEBUG    parser_guessit series:mark_as_collected Parsing result: <GuessitParsedSerie(name=Marvel's Inhumans (US),id=2,season=1,episode=2,quality=unknown,proper=0,status=OK)> (in 105.826 ms)
2017-11-04 17:03 DEBUG    parser        series:mark_as_collected FAIL: name regexps [u'^(?:(?:\\[[^\\[\\]]*\\])|(?:HD.720p?:)|(?:HD.1080p?:)|(?:HD.2160p?:))?(?:[^\\w&]|_)*(Marvel(?:[^\\w&]|_)*s(?:[^\\w&]|_)*Inhumans(?:[^\\w&]|_)*Us)(?:\\b|_)(?:[^\\w&]|_)*'] do not match Marvel's Inhumans - 1x02 - Those Who Would Destroy Us
2017-11-04 17:03 CRITICAL task          series:mark_as_collected BUG: Unhandled error in plugin metainfo_series: Series flagged invalid
2017-11-04 17:03 CRITICAL manager       series:mark_as_collected An unexpected crash has occurred. Writing crash report to /home/xxxxxx/.flexget/crash_report.2017.11.04.170301753480.log. Please verify you are running the latest version of flexget by using "flexget -V" from CLI or by using version_checker plugin at http://flexget.com/wiki/Plugins/version_checker. You are currently using version 2.10.104
2017-11-04 17:03 DEBUG    manager       series:mark_as_collected Traceback:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flexget/task.py", line 486, in __run_plugin
    return method(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/event.py", line 23, in __call__
    return self.func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/metainfo/series.py", line 32, in on_task_metainfo
    self.guess_entry(entry)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/metainfo/series.py", line 47, in guess_entry
    populate_entry_fields(entry, parsed, config)
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/filter/series.py", line 1307, in populate_entry_fields
    entry['series_id'] = parser.pack_identifier
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/parsers/parser_common.py", line 555, in pack_identifier
    return self.identifier
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/parsers/parser_common.py", line 546, in identifier
    return self.identifiers[0]
  File "/usr/local/lib/python2.7/dist-packages/flexget/plugins/parsers/parser_common.py", line 531, in identifiers
    raise Exception('Series flagged invalid')
Exception: Series flagged invalid
```

### Additional information:

- FlexGet version: 2.10.104
- guessit version: 2.1.4
- Python version: 2.7.13
- Using daemon (yes/no): yes and no (tried both)
- OS and version: Linux xxxxx 4.9.59-v7+ #1047 SMP Sun Oct 29 12:19:23 GMT 2017 armv7l GNU/Linux

**Comment by cvium** [comment]
Use the internal parser instead. Any guessit-related bugs are unlikely to be fixed. You or anyone else are welcome to open a pull request though.

**Comment by gligoran** [comment]
@cvium I did that as a workaround, but what's going on with guessit? Has it been abandoned? Is the internal parser better than guessit?

**Comment by cvium** [comment]
The guessit plugin maintainer seems to have abandoned it, and no one else really wants to touch that code as it's in pretty bad shape (near-incomprehensible).

**Comment by gligoran** [comment]
OK, thanks. I guess it's better to just use internal parsing.

**Comment by cvium** [comment]
I'm going to assume this issue would be fixed if the plugin was updated (issue #1804). Closing this.
